README.md

```diff
@@ -106,25 +113,111 @@ See an example configuration below:
 }
 ```
 
+See example configuration for `helm-latest:helm-repo-fqdn`:
+
+```
+{
+  "cilium": {
+    "versionSource": "helm-latest:https://helm.cilium.io",
+    "imageDenylist": [
+      "quay.io/cilium/operator",
+      "quay.io/cilium/startup-script"
+    ],
+    "helmCharts": {
+      "cilium/cilium": {
+        "chartConfig": {
+          "aws": {
+            "values": [
+              "eni.enabled=true"
+            ],
+            "kubeVersion": "1.24"
+          },
+          "azure":  {
+            "values": [
+              "azure.enabled=true"
+            ]
+          },
+          "generic": {
+            "values": [
+              "clustermesh.useAPIServer=true",
+              "envoy.enabled=true",
+              "hubble.ui.enabled=true",
+              "hubble.relay.enabled=true",
+              "hubble.enabled=true"
+            ]
+          },
+          "kubeversiononly": {
+            "kubeVersion": "1.28"
+          }
+        }
+      }
+    }
+  },
+  "epinio": {
+    "versionSource": "helm-latest:https://epinio.github.io/helm-charts",
+    "helmCharts": {
+      "epinio/epinio": {
+        "chartConfig": {
+          "generic": {
+            "values": [
+              "global.domain=myepiniodomain.org"
+            ]
+          }
+        }
+      }
+    }
+  },
+  "kubewarden": {                                                                                                                                                                                                                                                                                                             
+    "versionSource": "helm-latest:https://charts.kubewarden.io",
+    "helmCharts": {
+      "kubewarden/kubewarden-controller": {},
+      "kubewarden/kubewarden-defaults": {}
```

Comment:
a nit: This requires specifying charts as `$REPO/$CHART`, when `$REPO` is already the key for this entry in the dict. Charts should be specified as just the chart name; the tool can build the `$REPO/$CHART` arg required by the CLI from the dict key + the name of the chart.

```suggestion
      "kubewarden-controller": {},
      "kubewarden-defaults": {}
```